src/rgw/rgw_http_client.h

```diff
@@ -57,6 +58,10 @@ class RGWHTTPClient
     has_send_len = true;
   }
 
+  void set_insecure() {
```

Comment:
Thinking of renaming  to set_verify_ssl(flag), so that config variable can directly be passed, I'll push the updated version..